yazi-plugin/src/external/shell.rs

```diff
@@ -32,6 +32,15 @@ pub fn shell(opt: ShellOpt) -> Result<Child> {
 	#[cfg(unix)]
 	return Ok(unsafe {
 		Command::new("sh")
+			.env(
+				"YAZI_LEVEL",
+				(env::var("YAZI_LEVEL")
+					.ok()
+					.and_then(|lvl_str| lvl_str.parse::<i32>().ok())
```

Comment:
Can it be a negative number? If not we should use `u32` here.

Reply:
Technically it can, if the user manually sets it to be so, but I cannot think of any use case where that would actually make sense, so it's probably ok to go with `u32`. I pushed a `fixup` commit.